assume/reinforcement_learning/learning_utils.py

```diff
@@ -153,3 +163,140 @@ def func(_):
         return val
 
     return func
+
```

Comment:
I would rather put this in `assume/reinforcement_learning/tensorboard_logger.py`

eventually next to a `tensorboard_info.py` or just integrate the text into the same file then..? :)